Add passwordAuthenticationmethod and resetPassword to V1 (#17299)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/longrunningoperation.md

---

> title: "long-running operation resource type"

Reviewer: ```suggestion
title: "longRunningOperation resource type"
```

description: "The status of a long-running operation."
ms.localizationpriority: medium
author: "zhvolosh"
ms.prod: teamwork
doc_type: resourcePageType
---

> # Long running operation resource type

Reviewer: ```suggestion
# longRunningOperation resource type
```

Namespace: microsoft.graph

The status of a long-running operation.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |createdDateTime|DateTimeOffset|The start time of the operation.|
> |id|String|The unique identifier of the operation |
> |lastActionDateTime|DateTimeOffset|The time of the last action of the operation.|
> |resourceLocation|String| URI of the resource that is the operation is performed on. |
> |status|longRunningOperationStatus|The current status of the operation. The possible values are: `notStarted`, `running`, `succeeded`, `failed`, `unknownFutureValue`.|
> |statusDetail|String|Details on the status of the operation.|

Reviewer: ```suggestion
|createdDateTime|DateTimeOffset|The start time of the operation.|
|id|String|The unique identifier of the operation |
|lastActionDateTime|DateTimeOffset|The time of the last action in the operation.|
|resourceLocation|String| URI of the resource that the operation is performed on. |
|status|longRunningOperationStatus|The status of the operation. The possible values are: `notStarted`, `running`, `succeeded`, `failed`, `unknownFutureValue`.|
|statusDetail|String|Details about the status of the operation.|
```

## JSON representation

Here is a JSON representation of the resource.

> <!--{
>   "blockType": "resource",
>   "optionalProperties": [],
>   "baseType": "microsoft.graph.entity",
>   "@odata.type": "microsoft.graph.operation"
> }-->
> 
> ```json
> {
>   "createdDateTime": "String (timestamp)",
>   "lastActionDateTime": "String (timestamp)",
>   "status": "notStarted | running | completed | failed",
>   "statusDetail": "ResetSuccess", 
>   "id": "2d497bb-57bd-47a6-8749-5ccd0869f2bd",

Reviewer: Fix
```suggestion
<!-- {
  "blockType": "resource",
  "keyProperty": "id",
  "@odata.type": "microsoft.graph.longRunningOperation",
  "baseType": "microsoft.graph.entity",
  "openType": false
}
-->
``` json
{
  "@odata.type": "#microsoft.graph.longRunningOperation",
  "id": "String (identifier)",
  "createdDateTime": "String (timestamp)",
  "lastActionDateTime": "String (timestamp)",
  "resourceLocation": "String",
  "status": "String",
  "statusDetail": "String"
}
```
```
